R/estimate_R.R

```diff
@@ -28,59 +28,65 @@
 #' @param method One of "non_parametric_si", "parametric_si", "uncertain_si",
 #'   "si_from_data" or "si_from_sample" (see details).
 #'
+#' @param backimputation_window Length of the window used to impute incidence
```

Comment:
@sangeetabhatia03  should the new argument be added as last argument to ensure full back-compatibility in case people have not specified parameters i.e. are using this as estimate_R(a, b, c) rather than estimate_R(a = a, etc)?

Reply:
agree that would be ideal.

Reply:
@abriz97 can you please make that change? thanks